### ISSUE TITLE
Setting UILabel's paragraph style linespacing resets alignment to left

### ISSUE THREAD

**Issue by VilemKurz** [inloop/Styles#83]
Set a textstyle like this to a UILabel
```
TextStyle(.paragraphStyle([.lineSpacing(2.0)]))
```
The label has alignment set to center in storyboard.
BUG: The label has aligment left after applying the text style. In other words, setting line spacing has a side effect, it resets alignment to left.
EXPECTED BEHAVIOUR: After setting linespacing, the label should keep its alignment setting without any change.

And vice versa. If you set alignment only, line spacing is reset. So, it is not possible to do this: `TextStyle(.paragraphStyle([.alignment(.center)])) + TextStyle(.paragraphStyle([.lineSpacing(2.0)]))`

**Comment by jakubpetrik** [comment]
You have to provide also alignment. This is intended behaviour.

**Comment by VilemKurz** [comment]
I understand, this is how UIKit works. You set UILabel's alignment center, then attributed string with paragraph without alignment specified and boom, you have alignment left. But, shouldn't Styles shield us from these low level hassles?

**Comment by jakubpetrik** [comment]
No. This is about single source of truth. You set the style and expect it no side effects. In other words it should clear out everything and apply only the properties which are defined in the style. That way you can reason about it more easily. If this is not the case than its a bug that needs to be fixed.

**Comment by VilemKurz** [comment]
Ok, forget about mixing storyboard settings with Styles (although I think we agreed that Styles should preserve storyboard settings - in other words Styles should not change other values than set through Styles). You even can not do this, in a pure Styles fashion `TextStyle(.paragraphStyle([.alignment(.center)])) + TextStyle(.paragraphStyle([.lineSpacing(2.0)]))`
